lib/engineering/analysis/footing/centric_isolated.rb

```diff
@@ -0,0 +1,48 @@
+module Engineering
+  module Analysis
+    module Footing
+      class CentricIsolated
+        ORTOGONALITIES = %i[length_x length_y].freeze
+        CUT_DIRECTION_ERROR = 'Not valid cut direction, should one of thes: :length_x :length_y'.freeze
+
+        def initialize(column:, footing:, effective_height:, load_from_column:, cut_direction:)
+          @column = column
+          @footing = footing
+          @load_from_column = load_from_column
+          @effective_height = effective_height.to_f
```

Comment:
Since the Footing knows its height, its reinforcement and its location, do you think we can make this a method of the footer instead of an attribute here?

Reply:
Yeah, so with the last merged PR, now it was done